README.md

```diff
@@ -38,3 +47,22 @@ Handwrite can generate custom font of your handwriting based on a writing sample
 `python setup.py test`
 
 You are ready to go!
+
+# Usage
+
+## Creating your Handwritten Sample 
+
+1. Take a printout of the [sample form](https://github.com/cod-ed/handwrite/raw/main/handwrite_sample.pdf).
+2. Fill the form using the image below as a reference.
+3. Scan the filled form using a scanner, or Adobe Scan in your phone.
+4. Save the `.jpg` image in your system.
```

Comment:
Just a preference thing, can we leave blank line after each point, looks a little neat.